package.json

```diff
@@ -1,43 +1,27 @@
 {
   "name": "to-do-list-project",
   "version": "1.0.0",
-  "description": "",
-  "private": true,
+  "description": "<a name=\"readme-top\"></a>",
+  "main": "babel.config.js",
   "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1",
     "watch": "webpack --watch",
     "start": "webpack serve --open",
-    "build": "webpack"
-  },
-  "repository": {
-    "type": "git",
-    "url": "git+https://github.com/ClintonYounge/To-Do-List-Project.git"
+    "build": "webpack",
+    "test": "echo \"Error: no test specified\" && exit 1"
```

Comment:
- [ ] For proper documentation, in your README under the test section, kindly include the command for running the tests.

- [ ] When I run your tests with this command I get this error message. Kindly fix that. I believe "jest" is what you should instead not this error message.

  ![image](https://user-images.githubusercontent.com/36542986/223989384-e471df82-819b-449b-99fb-f2a43f9f26a8.png)